cromwell.template.conf

```diff
@@ -0,0 +1,147 @@
+include required(classpath("application"))
+google {
+  application-name = "cromwell"
+  auths = [
+    {
+      name = "application_default"
+      scheme = "application_default"
+    }
+  ]
+}
+engine {
+  filesystems {
+    gcs {
+      auth = "application_default"
+      project = "vlad-dev"
+    }
+  }
+}
+backend {
+  default = PAPIv2
+
+  providers {
+    PAPIv2 {
+      actor-factory = "cromwell.backend.google.pipelines.v2beta.PipelinesApiLifecycleActorFactory"
+      config {
+        # Google project
+        project = <project>
+
+        # Base bucket for workflow executions
+        root = "gs://<bucket>/cromwell/executions"
+
+        # Make the name of the backend used for call caching purposes insensitive to the
+        # PAPI version.
+        name-for-call-caching-purposes: PAPI
+
+        # Emit a warning if jobs last longer than this amount of time. This might indicate that
+        # something got stuck in PAPI.
+        slow-job-warning-time: 24 hours
+
+        # Number of workers to assign to PAPI requests
+        request-workers = 3
+
+        genomics {
+          # A reference to an auth defined in the `google` stanza at the top. This auth is used
+          # to create pipelines and manipulate auth JSONs.
+          auth = "application_default"
+
+          # Endpoint for APIs, no reason to change this unless directed by Google.
+          endpoint-url = "https://lifesciences.googleapis.com/"
+
+          # Currently Cloud Life Sciences API is available only in `us-central1` and
```

Comment:
It's a few more, maybe just put a link here?
https://cloud.google.com/life-sciences/docs/concepts/locations